clients/src/main/java/org/apache/kafka/common/network/ChannelBuilders.java

```diff
@@ -63,7 +64,7 @@ public static ChannelBuilder clientChannelBuilder(SecurityProtocol securityProto
                 throw new IllegalArgumentException("`clientSaslMechanism` must be non-null in client mode if `securityProtocol` is `" + securityProtocol + "`");
         }
         return create(securityProtocol, Mode.CLIENT, contextType, config, listenerName, false, clientSaslMechanism,
-                saslHandshakeRequestEnable, null, null);
+                saslHandshakeRequestEnable, null, null, TIME);
```

Comment:
This class uses the TIME variable in multiple places already, so I simply replicated the same solution.  If we wish to fix all of them then perhaps it can be resolved via a separate ticket as opposed to this KIP?

Reply:
Oops, sorry, you are correct.  Fixed.